### ISSUE TITLE
A11y_Narrator unnecessarily announcing blank cell to the check box present grid cell.

### ISSUE THREAD

**Issue by ShaileshChudawkar** [microsoft/azuredatastudio#25001]
### GitHub Tags:
 #A11yTCS; #A11ySev2; #DesktopApp; #Benchmark; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #FTP; #Narrator; #WCAG1.3.1; #ScreenReader; #Win11; #Win32; #AILimited;
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Actual Result:

Narrator unnecessarily announcing blank cell to the check box present grid cell.
### Expected Result:
Narrator should not announce blank cell to the check box cell.
### Environment Details:
Application Name: Oracle Data Migration extension for Azure Data Studio
Download application: Azure Data Studio.
Operating System: Window 11 version 22H2 OS build 25267.1000.
### Repro Steps:

1. Turn on Narrator
2.  Open Azure Data Studio application.
3. Install "Data Migration for Oracle"
4. connect to source connection.
5. Tab till new Migration and invoke on it.
6. Navigate to step 2: Select Schema(s)
7. Verify Narrator unnecessarily announcing blank cell to the check box present grid cell.

### User Impact:
If screen reader announces incorrect information user get difficulty to understand.

https://github.com/microsoft/azuredatastudio/assets/143179527/f651f4e2-9067-4e1b-b976-4874e20240ad

![Code snippet](https://github.com/microsoft/azuredatastudio/assets/143179527/eedab15f-e590-4bce-9021-78077ca5d52c)

https://github.com/microsoft/azuredatastudio/assets/143179527/d7fe60d0-4298-41c8-906e-8ce44807b312

**Comment by ShaileshChudawkar** [comment]
Primary bug: https://msdata.visualstudio.com/Migration%20Assistant/_workitems/edit/2663245

**Comment by kisantia** [comment]
@ShaileshChudawkar can you please clarify that this is only not behaving as expected for Narrator, but NVDA is reading the checkbox as expected? From my understanding on previous accessibility bugs, only one screenreader on a platform needs to read as expected.

**Comment by PadminiGariki** [comment]
@kisantia 
verified bug in below environment.
Application Name: Oracle Data Migration extension for Azure Data Studio
Download application: Azure Data Studio.
Operating System: Window 11 version 23H2

This issue is not repro with NVDA and Jaws. It only repro with narrator. We downgrade the severity to sev3. Jaws attachment is provided below. 
As this is Sev3 and this bug don't have any defined SLA. Your team can add them to their backlogs and can be fixed as per your convenience.

#Regressed:12-20-23; #A11ySev3;

### Attachment:

[Issue is not repro with Jaws.webm](https://github.com/microsoft/azuredatastudio/assets/142479541/ff94b5a1-6d94-4c28-95ab-d40ddffff00f)
